writer.go

```diff
@@ -375,10 +378,10 @@ func (w *Writer) WriteMessages(ctx context.Context, msgs ...Message) error {
 			break
 		}
 	}
-
-	t1 := time.Now()
-	w.stats.writeTime.observeDuration(t1.Sub(t0))
-
+	if !w.config.Async {
+		t1 := time.Now()
+		w.stats.writeTime.observeDuration(t1.Sub(t0))
```

Comment:
While this stat is not very meaningful in async mode, it's a bit weird that we just won't be reporting it anymore, some programs out there could be depending on it.

Would it be possible to keep it?